src/stan/io/reader.hpp

```diff
@@ -1186,55 +1186,56 @@ class reader {
   }
 
   template <typename TL>
-  inline vector_t vector_lb(const TL lb, size_t m) {
+  inline vector_t vector_lb(const TL lb, const size_t m) {
```

Comment:
You don't need `const` arguments on primitives like `size_t` as it doesn't affect the API because it's call by value.  

And if `TL` can be anything other than a primitive, it should be a reference, and if it has to be a primitive, it shouldn't have `const`.

This applies all the way down.

Reply:
Word! I'll change these back